specification/maps/data-plane/Search/preview/2022-09-01-preview/search.json

```diff
@@ -0,0 +1,966 @@
+      "type": "string",
+      "enum": [
+        "High",
+        "Medium",
+        "Low"
+      ],
+      "x-ms-enum": {
+        "name": "ConfidenceEnum",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "High"
+          },
+          {
+            "value": "Medium"
+          },
+          {
+            "value": "Low"
+          }
+        ]
+      }
+    },
+    "NextLink": {
+      "type": "string",
+      "description": "The is the link to the next page of the features returned. If it's the last page, no this field."
+    },
+    "FeatureType": {
+      "type": "string",
+      "enum": [
+        "FeatureCollection"
+      ],
+      "x-ms-enum": {
+        "name": "FeatureTypeEnum",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "FeatureCollection"
+          }
+        ]
+      }
+    },
+    "Boundary": {
+      "description": "`GeoJSON GeocodingFeature` object that describe the boundaries of a geographical area. Geometry of the feature is described with `GeoJSON GeometryCollection`.\n\nPlease note for the Boundary search Polygon API, the GeometryCollection Polygon and MultiPolygon are the only geometry objects expected in the main response for now.",
```

Comment:
Please note, the service typically returns a GeometryCollection with Polygon or Mulitpolygon sub-types.